HeroVault/HeroVault/Features/User/UserFavouritesScreen/UserFavouritesScreenViewController.swift

```diff
@@ -14,36 +14,47 @@ class UserFavouritesScreenViewController: UIViewController, ViewModelProtocol {
     @IBOutlet weak private var userImage: UIImageView!
     @IBOutlet weak private var favouritesTitleLabel: UILabel!
     @IBOutlet weak private var favouritesCollectionView: UICollectionView!
+    @IBOutlet weak private var noFavouritesLabel: UILabel!
     
     // MARK: Variables
     
     private lazy var userFavouritesScreenViewModel = UserFavouritesScreenViewModel(userFavouritesScreenRepository: UserFavouritesScreenRepository(),
                                                                                    delegate: self)
-    
     // MARK: Functions
     
     override func viewDidLoad() {
         super.viewDidLoad()
         setUpCollectionView()
         setUpScreen()
+        noFavouritesLabel.isHidden = !userFavouritesScreenViewModel.showFavouritesText
     }
     
     func setUpScreen() {
         userFavouritesScreenViewModel.fetchAllMarvelDataFromCoreData()
         favouritesTitleLabel.text = userFavouritesScreenViewModel.favouritesScreenTitle
-        
+        if userFavouritesScreenViewModel.marvelDataListCount > 0 {
```

Comment:
rather make a function in the viewmodel that returns true if marvelDataCount is emty or not